src/core/Akka/Actor/Props.cs

```diff
@@ -152,6 +221,7 @@ public Props(Deploy deploy, Type type, params object[] args)
         ///     Gets the type.
         /// </summary>
         /// <value>The type.</value>
+        [JsonIgnore]
```

Comment:
Json.NET includes readonly properties into the serialized json.
That is because Json.NET is a bit too smart for its own good, it can use those readoly properties as arguments to a parameterized constructor.

We now use the argless default ctor and so we only need data from properties that have a setter.